DetectorTomography.py

```diff
@@ -44,137 +44,46 @@ def gtc_matrix_product_counting_function(arguments: list):
     return result
 
 
-class DetectorTomographyFitter:
-    """
-        This class is meant to resemble qiskit's state tomography and process tomography fitters and to calculate the
-        maximum likelihood povm estimator describing a detector basing on QDT job results and used probe states.
-    """
+class QDTCalibrationSetup:
 
-    def __init__(self, algorithm_convergence_threshold=1e-6):
-        self.algorithmConvergenceThreshold = algorithm_convergence_threshold
+    def __init__(self, qubits_number: int, probe_kets: List[np.array], frequencies_array: np.ndarray ):
+        self.qubits_number = qubits_number
+        self.probe_kets = probe_kets
+        self.probe_states = self.__get_probe_states(qubits_number, probe_kets)
+        self.frequencies_array = frequencies_array
 
-    def get_maximum_likelihood_povm_estimator(self, results_list: List[Result],
-                                              probe_kets: List[np.array], qiskit_register_convention=True) \
-            -> List[np.ndarray]:
+    @classmethod
+    def from_qiskit_results(cls, results_list: List[Result], probe_kets: List[np.array]):
         """
         Description:
-            Given results of Quantum Detector Tomography experiments and list of probe states, return the Maximum
-            Likelihood estimation of POVM describing a detector. Uses recursive method from [1]. See also [2].
-
-        Parameters:
-            :param results_list: List of results obtained from executing qiskit jobs.
-            :param probe_kets: A set of probe kets used to perform tomography.
-            :param qiskit_register_convention: Qiskit register convention is reverse to what we usually work with.
-            This solves the problem.
-
-        Returns
-            Maximum likelihood estimator of POVM describing a detector.
+            This method generates Calibration setup objects directly from qiskit job results and probe kets used
+            to generate circuits for these jobs. This method should be interpreted as sort of additional constructor
+            for qiskit users.
```

Comment:
Agreed, but still, we don't have "frequencies_array" constructor for non-qiskit results, right? :-)

Reply:
Default constructor takes frequencies_array and some other data sa input, so I don't agree. We do have general constructor.